### ISSUE TITLE
setup instruction for ubuntu Linux

### ISSUE THREAD

**Issue by RockyLinuxEnthusiast** [danielgwilson/reddit-youtube-video-bot#7]
How to setup on Linux

**Comment by AliHaider20** [comment]
@danielgwilson This is great project, but can you add a usage instructions for it?

**Comment by danielgwilson** [comment]
I can if you like - it's pretty darn dated though, was considering refreshing but most of the stuff like the reddit scrape probably doesn't work anymore

**Comment by AliHaider20** [comment]
Yeah, document or instructions will be helpful. Also, if they are different for Linux and Windows
